### DISCUSSION TITLE
p/ea7937c3bf4a4db88509943583eeeff0/

### DISCUSSION THREAD

**Discussion by giscus** [rxliuli/blog#32] [Announcements]
# p/ea7937c3bf4a4db88509943583eeeff0/

由于吾辈自己的成长经历、信息渠道的局限性，可能存在一些错误的刻板印象和认知，请保持求同存异的宽容心态。  已经定下了 10 月份前往日本的计划，这是吾辈第一次出国，也是想要出去看看更大的世界，所以想谈谈整个想法是如何产生和实施的。 动机关于出国的动机，原因是多方面的，吾辈可以说 2022 年的疫情是引子，但更重要的还是为了寻找变化。 遭受铁拳说起遭受铁拳，总体而言吾辈挨了两次。一次是最近三年所有

https://blog.rxliuli.com/p/ea7937c3bf4a4db88509943583eeeff0/

**Comment by Young-Flash** [comment]
加油，期待你的好消息👍

**Comment by zhl111** [comment]
写的不错哦，无论怎样都是你独特生命历程的开始，加油

**Comment by Hanaasagi** [comment]
おめでとう(｡˃ ᵕ ˂ )🎉

**Comment by AsukaRyuko** [comment]
倒是羡慕了，我还在温饱线附近挣扎，月月光……
祝你可以顺利到达自己的理想国定居生活

**Reply by rxliuli** [reply]
来了快一个月后，吾辈可以说日本真的不是应许之地

**Comment by Zourunfa** [comment]
祝老哥一路平安

**Comment by miaopass-future** [comment]
语言留学需要前置学位加持么

**Reply by rxliuli** [reply]
不

**Comment by olojiang** [comment]
哈哈~ 每个人都是独立的，每个人的思想都不一致，可以不完全赞同彼此，但是很难不承认，可以从阿凯身上，学习和学到些东西~

**Comment by littleANT800** [comment]
加油！钦佩你的勇敢以及行动力

**Comment by hengshanMWC** [comment]
父母，兄弟都在国外，如果我英语好点就考虑出去了

**Reply by rxliuli** [reply]
语言总是能学的（

**Comment by wenqi73** [comment]
祝顺利
